README.md

```diff
@@ -139,6 +140,26 @@ $ remme account transfer-tokens \
 }
 ```
 
+### Public key
+
+Get list of the public keys by account address — ``remme public-key get-list``:
+
+| Arguments | Type   | Required | Description                                |
+| :-------: | :----: | :------: | ------------------------------------------ |
+| address   | String | Yes      | Address to get list of the public keys by. |
```

Comment:
Apply the comment above to the argument description.

Reply:
@dmytrostriletskyi, done.